docs/diagrams/StorageLoadSequence.puml

```diff
@@ -0,0 +1,47 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
```

Comment:
Unsure if footbox, the boxes below, are necessary as examples shown did not have them. Can include `!include style.puml` to remove boxes or `hide footbox`